### ISSUE TITLE
Add support for more audio tracks to be captured in recordings.

### ISSUE THREAD

**Issue by ryantheleach** [pkviet/obs-studio#50]
**Is your feature request related to a problem? Please describe.**
I'm somewhat frustrated by OBS's default audio channels of 6. I'm hoping to record live performances, and having 6 channels for post-editing isn't enough.

**Describe the solution you'd like**
8-10 audio tracks would be enough for my usecase, but it does seem odd that OBS doesn't allow as many as your system can handle.

**Describe alternatives you've considered**

Running an external audio recorder, and resyncing the video and audio up afterwards.

Recompiling OBS myself to add more channels.

**Comment by pkviet** [comment]
Yeah that's something I did consider.
The main problem is that the tracks are hard-coded everywhere in the UI of obs: so it's really shitty to add tracks because they're not coded as vectors (in which case the track numbers could be set by a one line change in the code).
In UI, currently there are custom qt widgets for each track for instance, e.g. track1widget, ... track6widget . The track number is hardcoded.
In summary it's a big big hassle which would require a big rewrite.
Since I don't use more than 6 tracks, there's 0 odds that I'll implement it. It's not worth the time for me, given it's a hobby project on which I earn mostly 0 money (or symbolic, like 0.1$ donations. I'm not kidding. I've had donations of that amount.)
